src/Control/Monad/Static.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE StaticPointers #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Control.Monad.Static where
+
+import Control.Applicative.Static
+import Control.Distributed.Closure
+import Data.Functor.Static
+import Data.Typeable (Typeable)
+
+-- | Instances of 'StaticBind' should satisfy the following laws (writing
+-- 'staticMap', 'staticApply', 'staticBind' as infix @('<$>')@, @('<*>')@, @(>>=)@,
+-- respectively):
+--
+-- @
+-- (m >>= f) >>= g = m >>= static (.) ``cap`` (staticFlippedBind g) `cap` f
```

Comment:
We can recover one from the other for any concrete instance, so I wouldn't worry too much about it. ;)